### DISCUSSION TITLE
Would it be possible to add L1 Distance and Chebyshev Distance to uvecs and ivecs?

### DISCUSSION THREAD

**Discussion by Veliladon** [bitshifter/glam-rs#414] [Ideas]
Hi,

I've been working on a game in Bevy using grids and the library is fantastic but I find myself often looking for either the l1 distance and Chebyshev distance based on points on my grid. Would it be possible to add cdistance and ldistance methods to uvec and ivec to more easily calculate them?

**Comment by bitshifter** [comment]
For reference there was a PR #415 that needed additional work to be accepted, for anyone who was interested in adding support for this the PR was most of the way there.

**Comment by bitshifter** [comment]
Added
